src/pages/fieldManagement/settings/defaultFieldComponents/Editors.jsx

```diff
@@ -3,40 +3,38 @@ import { get } from 'lodash-es';
 
 import TreeEditor from './TreeEditor';
 import ConfigureDefaultField from './ConfigureDefaultField';
+import RelationshipEditor from './RelationshipEditor';
 
-export function RelationshipEditor() {
-  return null;
-}
-
-// export function RelationshipEditor({
-//   onClose,
-//   onSubmit,
-//   formSettings,
-//   setFormSettings,
-//   ...rest
-// }) {
-//   const speciesOptions = get(formSettings, 'relationships', []);
+export function RelationshipEditorWrapper({
```

Comment:
so a future programmer could easily grab "RelationshipEditor" try to start working with it. might take someone a while to realize that they need to be using the wrapper and not the RelationshipEditor itself. for this reason i prefer something like 

```
RelationshipEditor
 |_ RelationshipEditorCore 
```

Reply:
I like this idea a lot. Thanks. Done.